packages/ui-scripts/package.json

```diff
@@ -18,32 +18,29 @@
   },
```

Comment:
I actually really dislike that `ui-scripts` has 3 differently named binaries. What do you think of keeping just one (e.g. `ui-test --randomize` will be `ui-scripts test --randomize`) ? (in a later PR)

Reply:
I think this is a very good idea. Lets have a task about it!

Reply:
👍 ill create one